modules/KIWIImage.pm

```diff
@@ -4125,9 +4125,17 @@ sub extractLinux {
     my $xml       = $this->{xml};
     my $vconf     = $xml -> getVMachineConfig();
     my $xendomain;
-    my $kernname = "vmlinuz";
-    if (-f "$imageTree/boot/vmlinux") {
-        $kernname = "vmlinux";
+    my $kernname;
+    foreach (
+        "image",   # s390x
+        "vmlinux", # ppc*
+        "Image",   # aarch64
+        "uImage",  # armv* (legacy)
+        "zImage",  # armv*
+        "vmlinuz", # x86*
+        ) {
```

Comment:
yes and that issue should not be present if the code in KIWIConfig.sh around line 1548 would have matched it. You should also see a log message ''Failed to find a mapping kernel' in the kiwi log file in this case